src/fpti.js

```diff
@@ -58,7 +58,8 @@ export const FPTI_KEY = {
     STICKINESS_ID:             ('stickiness_id' : 'stickiness_id'),
     TIMESTAMP:                 ('t' : 't'),
     OPTION_SELECTED:           ('optsel' : 'optsel'),
-    USER_IDENTITY_METHOD:      ('user_identity_method' : 'user_identity_method')
+    USER_IDENTITY_METHOD:      ('user_identity_method' : 'user_identity_method'),
+    FIELDS_SESSION_ID:         ('fields_session_id' : 'fields_session_id')
```

Comment:
Is this for the new Fields component for APMs? If so can we include "component" in the key name?
```suggestion
    FIELDS_COMPONENT_SESSION_ID:         ('fields_component_session_id' : 'fields_component_session_id')
```